doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -285,6 +285,7 @@ Libraries for networking
   * Added:
 
     * A public header file :file:`nrf_cloud_defs.h` that contains common defines for interacting with nRF Cloud and the :ref:`lib_nrf_cloud` library.
+    * A new event :c:enum:`NRF_CLOUD_EVT_TRANSPORT_CONNECT_ERROR` to indicate an error while the transport connection is being established when the :kconfig:option:`CONFIG_NRF_CLOUD_CONNECTION_POLL_THREAD` Kconfig option is enabled.
```

Comment:
I think we need to add something like:
Earlier this was indicated with second NRF_CLOUD_EVT_TRANSPORT_CONNECTING event with error status

Reply:
Yes, that's probably a good idea.